modules/nf-core/genomad/download/genomad-download.diff

```diff
@@ -0,0 +1,14 @@
+Changes in module 'nf-core/genomad/download'
+--- modules/nf-core/genomad/download/main.nf
++++ modules/nf-core/genomad/download/main.nf
+@@ -4,7 +4,7 @@
+     conda "bioconda::genomad=1.5.2"
+     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+         'https://depot.galaxyproject.org/singularity/genomad:1.5.2--pyhdfd78af_0':
+-        'biocontainers/genomad:1.5.2--pyhdfd78af_0' }"
++        'quay.io/biocontainers/genomad:1.5.2--pyhdfd78af_0' }"
```

Comment:
This is out of date, we should update all modules to use the new registries... but I'll makew an issue for this